docs/plugins/integrating-search-into-plugins.md

```diff
@@ -138,13 +155,109 @@ export class FaqCollatorFactory implements DocumentCollatorFactory {
+  pluginId: 'search',
+  moduleId: 'faq-snippets-collator',
+  register(env) {
+    env.registerInit({
+      deps: {
+        config: coreServices.rootConfig,
+        logger: coreServices.logger,
+        scheduler: coreServices.scheduler,
+        indexRegistry: searchIndexRegistryExtensionPoint,
+      },
+      async init({ config, logger, scheduler, indexRegistry }) {
+        const defaultSchedule = {
+          frequency: { minutes: 10 },
+          timeout: { minutes: 15 },
+          initialDelay: { seconds: 3 },
+        };
+
+        const schedule = config.has('faq.schedule')
+          ? readSchedulerServiceTaskScheduleDefinitionFromConfig(
+              config.getConfig('faq.schedule'),
+            )
+          : defaultSchedule;
+
+        indexRegistry.addCollator({
+          schedule: scheduler.createScheduledTaskRunner(schedule),
+          factory: FaqSnippetsCollatorFactory.fromConfig(config, { logger }),
+        });
+      },
+    });
+  },
+});
+```
+
+In the fragment above, the module is registered, and when the Backstage backend initializes it, it adds the FAQ Snippets collator to the search index registry. Now let's export the module as default from the `index.ts` file:
+
+```ts title='plugins/search-backend-module-faq-snippets-collator/src/index.ts'
+export { export const searchFaqSnippetsCollatorModule as default } from './module';
+```
+
+#### 7. Install the collator module
+
+The newly created module should be added to the backend package dependencies as follows:
+
+```sh
+yarn --cwd backend add @internal/backstage-plugin-search-backend-module-faq-snippets-collator
+```
+
+After that, install the module on your Backstage backend instance:
+
+```ts title='packages/backend/src/index.ts'
+import { createBackend } from '@backstage/backend-defaults';
+//...
+const backend = createBackend();
+// Installing the search backend plugin
+backend.add(import('@backstage/plugin-search-backend/alpha'));
+// Installing the newly created faq snippets collator module
+backend.add(
+  import(
+    '@internal/backstage-plugin-search-backend-module-faq-snippets-collator'
+  ),
+);
+//...
+backend.start();
+```
+
+#### 8. Testing the collator code
 
 To verify your implementation works as expected make sure to add tests for it. For your convenience, there is the [`TestPipeline`](https://backstage.io/docs/reference/plugin-search-backend-node.testpipeline) utility that emulates a pipeline into which you can integrate your custom collator.
 
 Look at [DefaultTechDocsCollatorFactory test](https://github.com/backstage/backstage/blob/de294ce5c410c9eb56da6870a1fab795268f60e3/plugins/techdocs-backend/src/search/DefaultTechDocsCollatorFactory.test.ts), for an example.
 
-#### 6. Make your plugins collator discoverable for others
+You can also check out the documentation on [how to test Backstage plugin modules](https://backstage.io/docs/backend-system/building-plugins-and-modules/testing).
```

Comment:
should this be a relative path instead of a link, like the others?

Reply:
Good catch, fixed 🙏🏻